components/atoms/Accordion2.js

```diff
@@ -0,0 +1,87 @@
+import propTypes from "prop-types";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
+
+export function Accordion2(props) {
+  const [openFlag, setOpenFlag] = useState(false);
+
+  function openSummary() {
+    setOpenFlag(!openFlag);
+  }
+
+  const arrowColor = "#335075";
+
+  return (
+    <div className="flex mb-0 ">
+      <details
+        className={`accordion flex-1 flex-wrap border border-gray-300 rounded bg-white ((${props.mainClass})? ${props.mainClass} : '') `}
+      >
+        <summary onClick={() => openSummary()}>
+          <div className="w-12 min-w-12 relative left-4 py-2 float-left ">
+            {openFlag ? (
+              <FontAwesomeIcon
+                icon={faChevronRight}
+                rotation={90}
+                size="md"
+                color={arrowColor}
+              />
+            ) : (
+              <FontAwesomeIcon
+                icon={faChevronRight}
+                size="md"
+                color={arrowColor}
```

Comment:
```suggestion
              <FontAwesomeIcon
                icon={faChevronRight}
                size="md"
                className="text-custom-blue-reportButton"
```
Just a suggestion rather than have a color variable, but both ways are fine (would just need to change the color variable to match the color hex of the accordion text